android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -212,7 +223,7 @@ public void onListObtained(boolean successful) {
      * @param altTransportWake if the alt transport flag should be set. Only used in debug
      * @param device           the connected bluetooth device
      */
-    private static void startRouterService(Context context, ComponentName componentName, boolean altTransportWake, BluetoothDevice device, boolean confirmedDevice) {
+    private static void startRouterService(Context context, ComponentName componentName, boolean altTransportWake, BluetoothDevice device, boolean confirmedDevice, VehicleType vehicleType) {
```

Comment:
@kostyaBoss For `startRouterService()` method I would add in the JavaDoc comments for the two new parameters `boolean confirmedDevice` and `VehicleType vehicleType`.